src/database/remove.ts

```diff
@@ -29,10 +29,11 @@ export default class DatabaseRemove {
    * @param instance RTBD instance ID.
    * @param path path to delete.
    * @param host db host.
+   * @param disableTriggers if true, suppresses any Cloud functions that would be triggered by this operation.
    */
-  constructor(instance: string, path: string, host: string) {
+  constructor(instance: string, path: string, host: string, disableTriggers: boolean = false) {
```

Comment:
I think we should not provide a default here, so that each instantiation of `DatabaseRemove` must specify `disableTriggers`.

It's not used in many places.

Reply:
Removed default.